### PR TITLE
Implement spc(special character replacing) module

### PR DESCRIPTION
**Pull request by OSsunNO**
Partially completed tasks:

1. define 'spc_dict' - initial version

2. Implement the logic to replace special character

3. Check the running time of spc module using test data ('seoul_freeboard.csv')

TODO:

1. update 'spc_dict'

2. Implement a logic to restore a converted value back to the original value

### CODE REVIEW

**Comment by OSsunNO** [comment]
- spc_dict file is not uploaded, so I uploaded it to our google drive (2023 Capstone)

- testing 1
   1) original data
image
   2) converted data
image
   3) module running time
image

- testing 2
   1) original data
image
   2) converted data
image
   3) module running time
image

File: dt-demo-be/modules/spc/run.py [hunk 1]
```diff
@@ -13,9 +33,16 @@ def run(fileName):
...
+                nf.write(target+'\n')
         f.close()
     nf.close()
-    
\ No newline at end of file
+    
+    # module processing end
+    end = time.time()
+    
+    print(f"{end-start:.5f} sec")
```
Comment (doorooful) [inline dt-demo-be/modules/spc/run.py:48]: Can you put description here?
i.e) f'Total running time of spc module: {end-start} sec'

**Review by doorooful — CHANGES_REQUESTED** [review]
I request little change on displaying time spent of the module! Also, can you upload dictionary xlsx file on PR comment?
Thanks for your work!!!

File: dt-demo-be/modules/spc/run.py [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
 import os
+import time
 
 def run(fileName):
-    print("---Special character substitution module processing---")
+    
+    print("---Special character replacing module processing---")
+    
+    # Read Excel and save to ../database/fake.txt
+    test_dict = pd.read_csv("/mnt/c/Users/user/VSProjects/CapstoneProject/database/seoul_freeboard.csv")
```
Comment (yoouung) [inline dt-demo-be/modules/spc/run.py:10]: Can you change the file path from absolute to relative?

File: dt-demo-be/modules/spc/run.py (hunk 1 above)
Comment (OSsunNO) [reply dt-demo-be/modules/spc/run.py:48]: output:
image

File: dt-demo-be/modules/spc/run.py (hunk 2 above)
Comment (OSsunNO) [reply dt-demo-be/modules/spc/run.py:10]: Modified code:
image

**Comment by doorooful** [comment]
[spc_dict_v1.xlsx](https://github.com/doorooful/CapstoneProject/files/12712577/spc_dict_v1.xlsx)

**Comment by yoouung** [comment]
LGTM!

**Pull request merged**